modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java

```diff
@@ -106,6 +108,34 @@ public NotificationQueueEntry addNotificationQueueEntry(
 					notificationRecipientSetting);
 		}
 
+		String subject = notificationQueueEntry.getSubject();
+
+		String body = notificationQueueEntry.getBody();
+
+		if (Pattern.compile(
+				"\\d{4}-\\d{2}-\\d{2} 00:00:00.0"
+			).matcher(
+				subject
+			).find() ||
+			Pattern.compile(
+				"\\d{4}-\\d{2}-\\d{2} 00:00:00.0"
+			).matcher(
+				body
+			).find()) {
+
+			String[] newSubject = subject.split("00:00:00.0");
+
+			String[] newBody = body.split("00:00:00.0");
+
+			notificationQueueEntry.setSubject(
+				StringUtil.merge(newSubject, " "));
+
+			notificationQueueEntry.setBody(StringUtil.merge(newBody, " "));
+
+			notificationQueueEntry = notificationQueueEntryPersistence.update(
+				notificationQueueEntry);
+		}
```

Comment:
Hi @vngoliveira,

I believe we should handle this case at [this place](https://github.com/liferay/liferay-portal/blob/33e52fb3e9302e3851a028995b3ec6ad42cc64ed/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java#L96-L126), because here we're already handling a specific case (Picklist) so we can adapt this method to fit our case, for example:

````java
	private Map<String, Object> _getTermValues(
		ObjectDefinition objectDefinition, Map<String, Object> variables) {

		Map<String, Object> termValues = (Map<String, Object>)variables.get(
			"baseModel");

		for (ObjectField objectField :
				_objectFieldLocalService.getObjectFields(
					objectDefinition.getObjectDefinitionId())) {

			if (termValues.get(objectField.getName()) == null) {
				continue;
			}

			if (Objects.equals(
				objectField.getBusinessType(),
				ObjectFieldConstants.BUSINESS_TYPE_DATE) &&
				!Objects.equals(objectField.getName(), "createDate") &&
				!Objects.equals(objectField.getName(), "modifiedDate")) {

				SimpleDateFormat simpleDateFormat =
					new SimpleDateFormat("yyyy-MM-dd");

				Timestamp timestamp =
					(Timestamp)termValues.get(objectField.getName());

				termValues.put(
					objectField.getName(),
					simpleDateFormat.format(new Date(timestamp.getTime())));
			}
			else if (Objects.equals(
				objectField.getBusinessType(),
				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {

				ListTypeEntry listTypeEntry =
					_listTypeEntryLocalService.fetchListTypeEntry(
						objectField.getListTypeDefinitionId(),
						(String)termValues.get(objectField.getName()));

				if (listTypeEntry != null) {
					termValues.put(
						objectField.getName(), listTypeEntry.getNameCurrentValue());
				}
			}
		}

		return termValues;
	}
````

let me know if makes sense to you, and you also need to adapt the integration tests [EmailNotificationType](https://testray.liferay.com/home/-/testray/case_results/2445734589) and [UserNotificationType](https://testray.liferay.com/home/-/testray/case_results/2445734591?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2445733129%26cur%3D1%26delta%3D200%26tab%3Druns%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc) because they're failing.

Thank you!